src/NuGet.Services.AzureSearch/DatabaseOwnerFetcher.cs

```diff
@@ -66,13 +71,17 @@ public async Task<string[]> GetOwnersOrEmptyAsync(string id)
                 var sortedOwners = owners
                     .OrderBy(o => o, StringComparer.OrdinalIgnoreCase)
                     .ToArray();
+
+                stopwatch.Stop();
+                _telemetryService.TrackGetOwnersForPackageId(sortedOwners.Length, stopwatch.Elapsed);
```

Comment:
You can use a `TrackDuration` here if you use `TrackDuration<TProperties>`. Example: https://github.com/NuGet/ServerCommon/pull/168

I don't feel strongly about this, your current code is succinct enough and doesn't require a new class to store the `sortedOwners`

Reply:
Same comment for `GetPackageIdToOwnersAsync`

Reply:
If there were mutable anonymous types this would be slam dunk. I don't think it's worth coming up with private classes for each of these cases.

Reply:
I'll try named tuple and see if it feels better

Reply:
Spoke offline. Decided against it.